pkg/services/grpcserver/interceptors/tracing.go

```diff
@@ -3,7 +3,10 @@ package interceptors
 import (
 	"context"
 
+	"go.opentelemetry.io/otel"
```

Comment:
Side note:

`otel` provides a pair of [interceptors](https://github.com/open-telemetry/opentelemetry-go-contrib/blob/9d4eb7e7706038b07d33f83f76afbe13f53d171d/instrumentation/google.golang.org/grpc/otelgrpc/interceptor.go#L327) that are essentially a direct superset of what's in this file.

Grafana has fully moved to otel now - all other `tracing.Tracer` implementations have been deleted from the codebase. We could consider also deleting this file entirely and using `otelgrpc.UnaryServerInterceptor`/`otelgrpc.StreamServerInterceptor` directly instead.